.github/scripts/generate_ci_workflows.py

```diff
@@ -293,6 +293,19 @@ def generate_workflow_file(self, workflow_template: jinja2.Template) -> None:
             labels={LABEL_CIFLOW_DEFAULT, LABEL_CIFLOW_CUDA, LABEL_CIFLOW_WIN}
         ),
     ),
+    CIWorkflow(
+        arch="windows",
+        build_environment="win-vs2019-cuda11.5-py3",
+        cuda_version="11.5",
+        test_runner_type=WINDOWS_CUDA_TEST_RUNNER,
+        num_test_shards=2,
+        only_run_smoke_tests_on_pull_request=True,
+        enable_force_on_cpu_test=1,
+        ciflow_config=CIFlowConfig(
+            run_on_canary=True,
+            labels={LABEL_CIFLOW_DEFAULT, LABEL_CIFLOW_CUDA, LABEL_CIFLOW_WIN}
```

Comment:
Can we make this a periodic job as well? And remove LABEL_CIFLOW_DEFAULT from here?

Reply:
Ah yes totally let me do this